user_docs/en/userGuide.t2t

```diff
@@ -2639,7 +2638,7 @@ Add-ons will be listed if the search text can be found in the display name, publ
 ++ Add-on actions ++[AddonStoreActions]
 Add-ons have associated actions, such as install, help, disable, and remove.
 The actions menu can be accessed for an add-on in the add-on list by pressing the ``applications`` key, ``enter``, right clicking or double clicking the add-on.
-There is also an Actions button in the selected add-on's details, which can be activated normally or by pressing ``alt+a``.
+There is also an Actions button in the selected add-on's details, which can be activated normally or by pressing ``alt+c``.

```

Comment:
If we are trying to remove accelerator keys:

```suggestion
There is also an Actions button in the selected add-on's details.
```

Reply:
maybe this should be merged with the previous sentence to be a list e.g.:

```
There is an actions button in the add-on details which opens a menu of actions.
The actions menu can also be accessed for an add-on in the add-on list by:
- pressing the ``applications`` key
- pressing ``enter``
- right clicking
- double clicking
```

Reply:
I have committed a minimal modification of the User Guide in 9b0f52fb4a87c68447ea20ced3ffbaa09992ab4a, just removing the accelerator key, making the assumption that users should know how to activate a button normally in Windows or discover shortcut keys through navigation.

I have then committed a wider rewording  of this paragraph (a5ce3ec4b6a1773a5b081f36c60cbf650305d621) that reads better IMO. Since it's less related to this PR, feel free to accept it (keeping a5ce3ec4b6a1773a5b081f36c60cbf650305d621) or reject it (I will revert a5ce3ec4b6a1773a5b081f36c60cbf650305d621 in this case).

In both cases, I have kept the context menu solution first since it should be the more straight forward and preferred solution.

Regarding the conversion to list of various way to access the menu, with a5ce3ec4b6a1773a5b081f36c60cbf650305d621, it does not seem necessary to me.